### ISSUE TITLE
Iam policy on bucket - not working

### ISSUE THREAD

**Issue by tomaszmacnar** [fsouza/fake-gcs-server#1257]
I use fake-gcs-server in my integration tests. 
I create bucket and after it i try to get IAm policy on it. After call (java): 

> storage.getIamPolicy("XXX", Storage.BucketSourceOption.requestedPolicyVersion(GCP_POLICY_VERSION));

and http return:
GET http://localhost:4443/storage/v1/b/XXX/iam?optionsRequestedPolicyVersion=3
Not Found
When i get bucket info it return OK:
GET http://localhost:4443/storage/v1/b/XXX

Is this mock not support iam ?

**Comment by fsouza** [comment]
Correct, that's currently not supported. With bucket attributes (#1174), it should be possible though.

**Comment by vitalii-buchyn-exa** [comment]
Hello community,

Having a similar issue, trying `IAM().Policy(ctx)` on a fake bucket, but getting 404

Just wonder if there is any workaround, except creating my own httptest server?

Thank you in advance.
